doc/index.rst

```diff
@@ -46,10 +46,25 @@ run ``M-x eval-buffer``:
 
    (global-flycheck-mode)
 
+*On MacOS* also add the following to :ref:`fix your $PATH environment variable
+<flycheck-macos-exec-path-from-shell>`:
+
+.. code-block:: cl
+
+   (package-install 'exec-path-from-shell)
```

Comment:
We're suggesting a `(package-refresh-contents)` in the code above, and it's the quickstart section. IOW, the user is supposed to type all that into scratch, in the same Emacs session, to try Flycheck.